lib/release/api.ts

```diff
@@ -1,42 +1,12 @@
 import pMap = require('p-map');
-import type { PinejsClientCore, RetryParameters } from 'pinejs-client-core';
-import { PinejsClientRequest } from 'pinejs-client-request';
+import type { PinejsClientCore } from 'pinejs-client-core';
 import * as models from './models';
 import type { Dict } from './types';
 
 import type { Composition } from '../../lib/parse';
 
 const MAX_CONCURRENT_REQUESTS = 5;
 
-export interface ClientConfig {
-	/**
-	 * The host address of the API server to use, complete with the protocol,
-	 * eg. `https://api.balena-cloud.com`. This module will issue requests to v4 of
-	 * the API.
-	 */
-	apiEndpoint: string;
-
-	/**
-	 * The complete string to forward as Authorization HTTP header, eg.
-	 * `Bearer <authtoken>`.
-	 */
-	auth: string;
-
-	retry?: RetryParameters;
-}
-
-export function createClient(config: ClientConfig): PinejsClientRequest {
```

Comment:
Now that this is gone, there is a comment at line 58/28 that needs to be updated.